ush/templates/FV3LAM_wflow.xml

```diff
@@ -378,6 +384,43 @@ MODULES_RUN_TASK_FP script.
 ************************************************************************
 -->
 
+  <metatask>
+    <var name="fhr"> {% for h in range(0, fcst_len_hrs+1) %}{{ " %03d" % h  }}{% endfor %} </var>
+
+    <task name="subset_and_ship_f#fhr#" maxtries=2>
+
+      &RSRV_SERVICE;
+      <command>&LOAD_MODULES_RUN_TASK_FP; "&SAVE_AND_SHIP_TN;" "&JOBSDIR;/JREGIONAL_SUBSET_AND_SHIP"</command>
+      <nodes>1:ppn=1</nodes>
+      <walltime>{{ wtime_upload }}</walltime>
+      <jobname>&SAVE_AND_SHIP_TN;{{ uscore_ensmem_name }}_f#fhr#</jobname>
+      <join><cyclestr>&LOGDIR;/&SAVE_AND_SHIP_TN;{{ uscore_ensmem_name }}_f#fhr#_@Y@m@d@H.log</cyclestr></join>
+
+      <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+      <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+      <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+      <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+      <envar><name>SLASH_ENSMEM_SUBDIR</name><value><cyclestr>{{ slash_ensmem_subdir }}</cyclestr></value></envar>
+      <envar><name>FHR</name><value>#fhr#</value></envar>
+      <envar><name>MEMBER</name><value>{{ uscore_ensmem_name }}</value></envar>
+
+      <dependency>
+        <or>
+          <taskdep task="&RUN_FCST_TN;{{ uscore_ensmem_name }}"/>
+          <and>
```

Comment:
The `<and>` should not be needed here, but it doesn't hurt anything.

Reply:
Oh yeah! I was copy/pasting from the old post job which needs the two netcdf files. Good catch.